### PR TITLE
fix(combobox): allow selected values to be updated through setValue

### LINKED ISSUE #1166: SetValue() of FormControl does not update displayed content of Combo box

### PR DESCRIPTION
**Pull request by Donisius**
Closes #1166

### CODE REVIEW

File: src/combobox/combobox.component.ts [hunk 1]
```diff
@@ -377,7 +377,6 @@ export class ComboBox implements OnChanges, AfterViewInit, AfterContentInit {
 				}
 				this.selected.emit(event);
 			});
-			this.view.items = this.items;
```
Comment (cal-smith) [inline src/combobox/combobox.component.ts]: How does removing this allow items to be updated via `setValue`/`writeValue`?

**Comment by cal-smith** [comment]
Everything else looks good BTW, I'm just curious what effect removing that one line has?

File: src/combobox/combobox.component.ts (hunk 1 above)
Comment (Donisius) [reply src/combobox/combobox.component.ts:380]: It already was updating the values but this line has been overwriting the values that are set through set/write value

Comment (cal-smith) [reply src/combobox/combobox.component.ts:380]: Interesting 🤔

**Pull request merged**